netlify.toml

```
@@ -4,24 +4,24 @@
 # DO NOT REMOVE THIS (contact @kubernetes/sig-docs-leads)
 publish = "public"
 functions = "functions"
-command = "make non-production-build"
+command = "cd themes/docsy && git submodule update -f --init && cd ../.. && make non-production-build"
 
 [build.environment]
-HUGO_VERSION = "0.59.1"
+HUGO_VERSION = "0.59.1 "
 
 [context.production.environment]
 HUGO_BASEURL = "https://kubernetes.io/"
 HUGO_ENV = "production"
 HUGO_ENABLEGITINFO = "true"
 
 [context.deploy-preview]
-command = "make deploy-preview"
+command = "cd themes/docsy && git submodule update -f --init && cd ../.. && make deploy-preview"
```

Do we want to update docsy every time or is it better to control when to accept docsy updates?

---

@kbhawkey I suspect it's better to take whatever pain comes with automatic updates--but I'm agnostic. Do you have an opinion one way or another? @sftim @onlydole?

---

I’ve been bit a few times by not pinning something to a specific version, and would like if we could do so here so we can have a guarantee of consistency. Otherwise we might have some hard to pin down bugs or results.

---

I missed this in my other review, but I recommend the same syntax here as elsewhere:
```suggestion
command = "git submodule update --init --recursive && make deploy-preview"
```